playbooks/roles/homebrew/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+- name: Check if Homebrew is installed
+  stat: path=/usr/local/bin/brew
+  register: homebrew_installed
+
+- name: Install Homebrew
```

Comment:
This task usually won't need to run in Packer builds, since we install Homebrew first via Packer in order to install Ansible. That only works because the vanilla VM seems to have the Xcode CLI tools package installed in it, otherwise Homebrew install I believe would fail.

Reply:
👍🏻